core/test/test_container.cpp

```diff
@@ -649,23 +649,23 @@ TEST(Task, timeout) {
 	t.add(std::make_unique<TimedForwardMockup>(timeout));
 
 	// no timeout, but limited number of solutions
-	EXPECT_TRUE(t.plan(3).val == moveit_msgs::MoveItErrorCodes::SUCCESS);
+	EXPECT_TRUE(t.plan(3));
 	EXPECT_EQ(t.solutions().size(), 3u);
 
 	// zero timeout fails
 	t.reset();
 	t.setTimeout(0.0);
-	EXPECT_TRUE(t.plan().val == moveit_msgs::MoveItErrorCodes::TIMED_OUT);
+	EXPECT_FALSE(t.plan());
```

Comment:
Here, the original check was more specific! Please revert.

Reply:
This isn't addressed yet, is it?

Reply:
Hm. It's not reverted in https://github.com/ros-planning/moveit_task_constructor/pull/319/commits/92a6267b80f1eab68987e4ca028867278b03a7aa, but it is actually fixed.